docs/integrations/sources/freshdesk.md

```diff
@@ -53,6 +53,7 @@ Please read [How to find your API key](https://support.freshdesk.com/support/sol
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+| 0.2.9 | 2021-11-16 | [8017](https://github.com/airbytehq/airbyte/pull/8017) | Maximum pagination limit "maximum_page = 500" is removed. |
```

Comment:
Rewording it to be more relevant to our target personas, specifically analytics engineers
```suggestion
| 0.2.9 | 2021-11-16 | [8017](https://github.com/airbytehq/airbyte/pull/8017) | Bugfix an issue that caused the connector not to sync more than 50000 contacts |
```